### PR TITLE
Issue #1496462 by acrollet, sun, cosmicdreams, Niklas Fiekas, aspilicious: convert RSS publishing settings to configuration system.

### PR DESCRIPTION
**Pull request by hellomrcat**
PR for https://github.com/backdrop/backdrop-issues/issues/170

### CODE REVIEW

File: core/modules/system/system.admin.inc [hunk 1]
```diff
@@ -1768,31 +1768,40 @@ function system_image_toolkit_settings() {
...
+      'fulltext' => t('Full text'),
+    ),
     '#description' => t('Global setting for the default display of content items in each feed.')
   );
 
-  return system_settings_form($form);
+  $form['submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Submit')
+  );
```
Comment (quicksketch) [inline core/modules/system/system.admin.inc:1848]: This needs the $form['actions']['#type'] ='actions' wrapper and putting the submit button inside of it.

File: core/modules/system/system.install [hunk 2]
```diff
@@ -1856,6 +1856,24 @@ function system_update_1011() {
 }
 
 /**
+ *  Convert RSS publishing settings to use configuration files.
+ */
+function system_update_1012() {
```
Comment (quicksketch) [inline core/modules/system/system.install:1862]: Now needs to be update_1014() I think.

**Comment by quicksketch** [comment]
This looks great, just one minor issue and needs a merge with 1.x.

**Comment by hellomrcat** [comment]
@quicksketch done

**Comment by quicksketch** [comment]
Looks like the change to `$form['submit']` didn't make it into the new update, maybe you missed a file?

**Comment by hellomrcat** [comment]
@quicksketch sorry I totally missed that note, should be fixed now.

**Pull request merged**